### PR TITLE
Add logo, favicon and footer copy

### PR DESCRIPTION
**Pull request by federicobucchi**
**Motivation**:

Since there is an existing logo, I thought about using it in the UI (on the header and favicon).

**Modifications**:

Changes on the header and footer.

**Visual Changes**:

Screenshot 2024-11-19 at 11 00 32 PM

---

Screenshot 2024-11-19 at 11 06 48 PM

### CODE REVIEW

**Comment by federicobucchi** [comment]
It succeed in local:

```
Configuration on demand is an incubating feature.

> Configure project :
This machine doesn't have any pre-configured repositories, will use: MavenRepo, Gradle Central Plugin Repository

> Task :servicetalk-grpc-health:pmdMain
Removed misconfigured rule: LoosePackageCoupling  cause: No packages or classes specified

> Task :servicetalk-grpc-health:pmdTest
Removed misconfigured rule: LoosePackageCoupling  cause: No packages or classes specified

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.8/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 6s
1383 actionable tasks: 113 executed, 1270 up-to-date
```

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -31,6 +31,14 @@ jobs:
...
         with:
           languages: ${{ matrix.language }}
+          extraction-options: >
+            {
+              "exclude": [
+                "**/*.ico",
+                "**/*.png",
+                "**/servicetalk-docs-gen.ipr"
+              ]
+            }
```
Comment (idelpivnitskiy) [inline .github/workflows/codeql-analysis.yml:41]:
```suggestion
```
I made a change for CodeQL in #3121, let's see if it helps.

File: servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkCorePlugin.groovy [hunk 2]
```diff
@@ -90,7 +90,7 @@ class ServiceTalkCorePlugin implements Plugin<Project> {
         source = fileTree(".") {
           includes = ["gradle/**", "*.gradle", "*.properties", "scripts/**", "buildSrc/**", "docs/**"]
           excludes = ["**/gradle/wrapper/**", "**/build/**", "**/.gradle/**", "**/gradlew*", "**/.cache/**",
-                      "**/*.svg", "**/.out/**", "**/node_modules/**", "**/*.png", "**/*.zip"]
+                      "**/*.svg", "**/.out/**", "**/node_modules/**", "**/*.png", "**/*.ico", "**/*.zip", "**/servicetalk-docs-gen.ipr"]
```
Comment (idelpivnitskiy) [inline servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkCorePlugin.groovy:93]:
```suggestion
                      "**/*.svg", "**/.out/**", "**/node_modules/**", "**/*.png", "**/*.ico", "**/*.zip", "**/*.ipr"]
```

**Review by idelpivnitskiy — APPROVED** [review]
Idk what's wrong with CodeQL. This PR is definitely good and should not be blocked. I proposed to merge as-is @Scottmitch.

@federicobucchi thank you for the contribution!

**Review by Scottmitch — APPROVED** [review]

**Pull request merged**